CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.6.1
+
+- Makes the retryable_request method also retry when a `RDStation::Error::Unauthorized` error happen
```

Comment:
Hi @Hilderlan, can you explain why retrying on this case is necessary?

Reply:
It was because the update wasn't happening when a token expired to call the email API client we recently added, but I think it was because we changed the authentication to be handled in Kong on some routes, so I'll try to refactor this PR to cover this case. 😁